rust/capture/src/limiters/redis.rs

```diff
@@ -74,98 +67,67 @@ impl RedisLimiter {
         interval: Duration,
         redis: Arc<dyn Client + Send + Sync>,
         redis_key_prefix: Option<String>,
+        resource: QuotaResource,
```

Comment:
move QuotaResource here instead of in the `is_limited` check, as a limiter is only ever used for a single resource type (we either run in Events more or Recordings mode, never both. If we did run both, we'd have two limiters not one shared one)

Reply:
Nice